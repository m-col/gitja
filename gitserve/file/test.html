<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="x-ua-compatible" content="ie=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>üêô gitserve - test</title>
    <link rel="stylesheet" type="text/css" href="https://gitserve.mcol.xyz/style.css"/>
    <link rel="icon" type="image/png" href="/favicon.png"/>
  </head>

  <body>
    <a id="heada" href="https://gitserve.mcol.xyz">
      <header>
        <h1 id="title">gitserve</h1>
        <h2 id="desc">üêô Templated web page generator for your git repositories</h2>
      </header>
    </a>

    <div id="nav">
      <span id="url">git clone <a href="https://github.com/m-col/gitserve">https://github.com/m-col/gitserve</a></span>

      <span id="links">
        <a href="https://gitserve.mcol.xyz">Home</a> |
        <a href="https://gitserve.mcol.xyz/gitserve/log.html">Log</a> |
        <a href="https://gitserve.mcol.xyz/gitserve/tree.html">Files</a> |
        <a href="https://gitserve.mcol.xyz/gitserve/refs.html">Refs</a> |
        <a href="https://gitserve.mcol.xyz/gitserve/file/README.rst.html">Readme</a> |
        <a href="https://gitserve.mcol.xyz/gitserve/file/LICENSE.html">License</a>
      </span>
    </div>

    <div id="content">
      <p>drwxr-xr-x test</p><hr>

<table id="files">
  <thead>
    <tr>
      <td><b>Mode</b></td>
      <td><b>Name</b></td>
    </tr>
  </thead>

  <tbody>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.config.dhall.html">test/config.dhall</a></td>
    </tr>
        <tr>
      <td>drwxr-xr-x</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.expected.html">test/expected</a></td>
    </tr>
        <tr>
      <td>drwxr-xr-x</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.expected.gitserve.html">test/expected/gitserve</a></td>
    </tr>
        <tr>
      <td>drwxr-xr-x</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.expected.gitserve.commit.html">test/expected/gitserve/commit</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.expected.gitserve.commit.0292014748caae952bbc8dd6225680d83c0a5135.html.html">test/expected/gitserve/commit/0292014748caae952bbc8dd6225680d83c0a5135.html</a></td>
    </tr>
        <tr>
      <td>drwxr-xr-x</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.expected.gitserve.file.html">test/expected/gitserve/file</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.expected.gitserve.file.test.expected.gitserve.file.html.html">test/expected/gitserve/file/test.expected.gitserve.file.html</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.expected.gitserve.file.test.templates.style.css.html.html">test/expected/gitserve/file/test.templates.style.css.html</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.expected.gitserve.index.html.html">test/expected/gitserve/index.html</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.expected.index.html.html">test/expected/index.html</a></td>
    </tr>
        <tr>
      <td>l---------</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.expected.link.html.html">test/expected/link.html</a></td>
    </tr>
        <tr>
      <td>drwxr-xr-x</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.expected.static.html">test/expected/static</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.expected.static.a_nice_file.html">test/expected/static/a_nice_file</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.expected.style.css.html">test/expected/style.css</a></td>
    </tr>
        <tr>
      <td>drwxr-xr-x</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.templates.html">test/templates</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.templates.index.html.html">test/templates/index.html</a></td>
    </tr>
        <tr>
      <td>l---------</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.templates.link.html.html">test/templates/link.html</a></td>
    </tr>
        <tr>
      <td>drwxr-xr-x</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.templates.repo.html">test/templates/repo</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.templates.repo.commit.html.html">test/templates/repo/commit.html</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.templates.repo.file.html.html">test/templates/repo/file.html</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.templates.repo.index.html.html">test/templates/repo/index.html</a></td>
    </tr>
        <tr>
      <td>drwxr-xr-x</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.templates.static.html">test/templates/static</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.templates.static.a_nice_file.html">test/templates/static/a_nice_file</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.templates.style.css.html">test/templates/style.css</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.templates.title.html.include.html">test/templates/title.html.include</a></td>
    </tr>
        <tr>
      <td>-rwxr-xr-x</td>
      <td><a href="https://gitserve.mcol.xyz/gitserve/file/test.test.sh.html">test/test.sh</a></td>
    </tr>
      </tbody>
</table>
</pre>
    </div>
  </body>
</html>
