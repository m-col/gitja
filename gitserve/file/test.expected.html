<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="x-ua-compatible" content="ie=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>üêô gitserve - test/expected</title>
    <link rel="stylesheet" type="text/css" href="http://gitserve.mcol.xyz/style.css"/>
    <link rel="icon" type="image/png" href="/favicon.png"/>
  </head>

  <body>
    <a id="heada" href="http://gitserve.mcol.xyz">
      <header>
        <h1 id="title">gitserve</h1>
        <h2 id="desc">üêô Templated web page generator for your git repositories</h2>
      </header>
    </a>

    <div id="nav">
      <span id="url">git clone <a href="https://github.com/m-col/gitserve">https://github.com/m-col/gitserve</a></span>

      <span id="links">
        <a href="http://gitserve.mcol.xyz">Home</a> |
        <a href="http://gitserve.mcol.xyz/gitserve/log.html">Log</a> |
        <a href="http://gitserve.mcol.xyz/gitserve/tree.html">Files</a> |
        <a href="http://gitserve.mcol.xyz/gitserve/refs.html">Refs</a> |
        <a href="http://gitserve.mcol.xyz/gitserve/file/README.rst.html">Readme</a> |
        <a href="http://gitserve.mcol.xyz/gitserve/file/LICENSE.html">License</a>
      </span>
    </div>

    <div id="content">
      <p>drwxr-xr-x test/expected</p><hr>

<table id="files">
  <thead>
    <tr>
      <td><b>Mode</b></td>
      <td><b>Name</b></td>
    </tr>
  </thead>

  <tbody>
        <tr>
      <td>drwxr-xr-x</td>
      <td><a href="http://gitserve.mcol.xyz/gitserve/file/test.expected.gitserve.html">test/expected/gitserve</a></td>
    </tr>
        <tr>
      <td>drwxr-xr-x</td>
      <td><a href="http://gitserve.mcol.xyz/gitserve/file/test.expected.gitserve.commit.html">test/expected/gitserve/commit</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="http://gitserve.mcol.xyz/gitserve/file/test.expected.gitserve.commit.0292014748caae952bbc8dd6225680d83c0a5135.html.html">test/expected/gitserve/commit/0292014748caae952bbc8dd6225680d83c0a5135.html</a></td>
    </tr>
        <tr>
      <td>drwxr-xr-x</td>
      <td><a href="http://gitserve.mcol.xyz/gitserve/file/test.expected.gitserve.file.html">test/expected/gitserve/file</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="http://gitserve.mcol.xyz/gitserve/file/test.expected.gitserve.file.test.expected.gitserve.file.html.html">test/expected/gitserve/file/test.expected.gitserve.file.html</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="http://gitserve.mcol.xyz/gitserve/file/test.expected.gitserve.file.test.templates.style.css.html.html">test/expected/gitserve/file/test.templates.style.css.html</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="http://gitserve.mcol.xyz/gitserve/file/test.expected.gitserve.index.html.html">test/expected/gitserve/index.html</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="http://gitserve.mcol.xyz/gitserve/file/test.expected.index.html.html">test/expected/index.html</a></td>
    </tr>
        <tr>
      <td>l---------</td>
      <td><a href="http://gitserve.mcol.xyz/gitserve/file/test.expected.link.html.html">test/expected/link.html</a></td>
    </tr>
        <tr>
      <td>drwxr-xr-x</td>
      <td><a href="http://gitserve.mcol.xyz/gitserve/file/test.expected.static.html">test/expected/static</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="http://gitserve.mcol.xyz/gitserve/file/test.expected.static.a_nice_file.html">test/expected/static/a_nice_file</a></td>
    </tr>
        <tr>
      <td>-rw-r--r--</td>
      <td><a href="http://gitserve.mcol.xyz/gitserve/file/test.expected.style.css.html">test/expected/style.css</a></td>
    </tr>
      </tbody>
</table>
</pre>
    </div>
  </body>
</html>
